utils/build_swift/driver_arguments.py

```diff
@@ -1026,6 +1036,17 @@ def create_argument_parser():
     option('--skip-test-optimize-none-with-implicit-dynamic', unsupported)
     option('--skip-test-optimized', unsupported)
 
+    # -------------------------------------------------------------------------
+    in_group('Build-script-impl arguments (for disambiguation)')
+    # We need to list --skip-build-swift explicitly because otherwise argparse
```

Comment:
Should this be "skip-**test**-swift" ?

Reply:
Yeah it should be. I’ll fix it in a follow-up PR.